roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -15,6 +15,10 @@ ExecStartPre=-/bin/mkdir -p /var/lib/kubelet
 EnvironmentFile={{kube_config_dir}}/kubelet.env
 # stage1-fly mounts /proc /sys /dev so no need to duplicate the mounts
 ExecStart=/usr/bin/rkt run \
+{% if rbd_mounts == true %}
```

Comment:
Can we not use a new var? Let's use `kubelet_load_modules`

Reply:
We can I just thought it was cleaner not to group all kinds of changes behind the same var, but It suits me